EIPS/eip-2794.md

```diff
@@ -0,0 +1,121 @@
+
+## Conclusion
+Our proposal presents a solution for mass exit of validators and eliminates two consequences that will cause devastating damages to ETH 2.0.
+- Active validators are tempted to exit and sell staked ETHs due to ETH's price hikes.
+- Active validators will get rewards when they perform an exit. Their rewards will not be associated with ETH's price hikes. Thus validators will not be tempted to speculate on ETH and will not be tempted to perform an exit for this speculation. 
+ETH's price hikes will not affect the real cost (the number of ETHs) for potential validators. ETH's price will not be positively associated with the cost of staking. In the long term a validator's income will increase and more people will likely join staking but the number of ETHs needed for staking for a validator will decrease. As more validators serve in the system the aforementioned extreme scenario will less likely happen.
+This proposal motivates more people to be validators and enhances validators' loyalty(work and behave properly) by rewarding validators with honor tokens. These honor tokens (including their logo images) will be permanently saved on Ethereum. This is the first ever proposal to reward Ethereum's contributors with honor and will greatly incentivize validators.
+
+Recently, a report "Ethereum 2.0 Economic Review" published on ConsenSys reveals some potential issues and risks with ETH 2.0 staking.
+
+- Security of the network in Eth2 is dependent upon three key variables: ETH staked, the price of ETH, and volatility.
+- Attacks on Eth2 are easier to scale than on Eth1. 
+- Targeting 13.8% ETH staked will match security levels of Eth1 at historical prices.
+- Economies of Scale for validating exist but are reduced at higher ETH prices.
+- Network security is heavily reliant on the price stability of ETH.
+- Beware of Derivative attacks. 
+Through our deep-dive analysis, each of these issues and risks is directly tied to ETH's prices which are affected/manipulated by trading activities in centralized crypto exchanges. In essence, if the improper staking mechanism is not redesigned these issues and risks might be triggered by ETH's price fluctuations. And our proposal completely eliminates the root cause of these issues and risks.
+
+In addition, our proposal addresses the following two points mentioned in this report and provides solutions.
+
+- While participation from Ethereum enthusiasts is important for a successful Beacon Chain launch, it is ultimately inadequate to reach sufficient levels of security. Attracting capital efficient validators will lead to higher fidelity in targeting a sufficient level of ETH staked.
+- This is a compelling serviceable addressable market, and a key objective of the incentive program to maximize network participation should be to convert these wallets into active validators.
+
+All in all, this proposal provides a solution for security and safety of ETH 2 Staking.
```

Comment:
Missing security section and copyright section from https://raw.githubusercontent.com/ethereum/EIPs/master/eip-template.md